test/bdd/features/orb-stress.feature

```diff
@@ -0,0 +1,13 @@
+
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
```

Comment:
Do you want this to run during regular orb builds?

Reply:
yes

Reply:
just 10 requests